Mathlib/Algebra/FreeAlgebra.lean

```diff
@@ -282,7 +282,7 @@ instance {R S A} [CommSemiring R] [CommSemiring S] [CommSemiring A]
     rw [smul_assoc, ←smul_one_mul]
 
 instance {R S A} [CommSemiring R] [CommSemiring S] [CommSemiring A]
-    [Algebra R A] [Algebra S A] [SMulCommClass R S A] :
```

Comment:
Or maybe I should ask, what happened here? Did the unused argument linter get beefed up?